doc/user/content/release-notes.md

```diff
@@ -55,6 +55,23 @@ Wrap your release notes at the 80 character mark.
 - Generate a persistent, unique identifier associated with each
   cluster. This can be retrieved using a new `mz_cluster_id()` SQL function.
 
+<span id="v0.4.3"></span>
+## v0.4.3 (Unreleased)
+
```

Comment:
If these are the notes for 0.4.3, should the "(unreleased)" be struck?

Reply:
Not yet! That will be @benLogN's job when he pulls the trigger on the release.